selfdrive/updated.py

```diff
@@ -302,11 +311,35 @@ def attempt_update():
 
     finalize_from_ovfs_copy()
 
+    # If a NEOS update is required, download it in the background
+    with open("/VERSION", "r") as current_neos_file:
+      current_neos_version = current_neos_file.read().replace("\n", "")
+    required_neos_version = run(["bash", "-c", r"source launch_env.sh && echo -n $REQUIRED_NEOS_VERSION"], FINALIZED)
+    if current_neos_version != required_neos_version or DEBUG_FORCE_UPDATE:
+      print(f"Beginning background download for NEOS {required_neos_version}")
+      try:
```

Comment:
why not `os.path.isdir`?

Reply:
I don't follow?

Reply:
```
if os.path.isdir(shutil.rmtree("/data/neosupdate")):
  shutil.rmtree("/data/neosupdate")
```

Reply:
Ah, didn't understand from the snippet github showed me of your question. Sure, change made.